pkg/cmd/roachtest/test_impl.go

```diff
@@ -59,8 +62,12 @@ type failure struct {
 type testImpl struct {
 	spec *registry.TestSpec
 
-	cockroach          string // path to main cockroach binary
-	cockroachShort     string // path to cockroach-short binary compiled with --crdb_test build tag
+	cockroach      string // path to main cockroach binary
+	cockroachShort string // path to cockroach-short binary compiled with --crdb_test build tag
```

Comment:
Nit: `cockroachAssertionsEnabled` (or similar, it seems elsewhere we use `cockroachEA` for "enabled assertions") is a better name than `cockroachShort`. Yes, the comment explains the real difference between these two things, but everyone sees the variable name, not everyone sees the comment.